source/supported-hardware/cml-rvp.rst

```diff
@@ -0,0 +1,125 @@
+.. _comet-lake-rvp:
+
+Comet Lake CRB Boards
+-----------------------
+
+.. note:: 10th Generation Intel® Core Processor is formally known as |Comet Lake S|.
+
+Supported Boards
+^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| supports **CML S RVP with PCH H, CML S RVP with PCH V, CML S RVP with H420E** platform.

```

Comment:
platform -> platforms

Reply:
I have corrected it in latest updated patch.